pkg/controller/operator/seed/resources/kubermatic/webhooks.go

```diff
@@ -367,3 +368,56 @@ func OperatingSystemProfileValidatingWebhookConfigurationCreator(ctx context.Con
 		}
 	}
 }
+
+func ApplicationDefinitionValidatingWebhookConfigurationCreator(ctx context.Context, cfg *kubermaticv1.KubermaticConfiguration, client ctrlruntimeclient.Client) reconciling.NamedValidatingWebhookConfigurationCreatorGetter {
```

Comment:
For me it is fine that this code is duplicated for **master**/.../webhooks.go and **seed**/.../webhooks.go and I think extracting the few lines into a separate package might be a bit overkill. Plus I guess it ties in with with Gos "duplication is far cheaper than the wrong abstraction" philosophy. But I would recommend that in this case we write a short comment on both funcs that their functionality is mirrored, so future devs are aware of it.

Reply:
I wanted them to be isolable resources but now that I think of it, it's better to have them in a separate shared package. We already do this for certain shared resources so it's completely fine to follow the same here.